nyxl-theme/src/css/_aui_custom.scss

```diff
@@ -0,0 +1,89 @@
+@font-face {
+	font-family: "Circular";
+	src: url("../fonts/Circular/Circular-Medium.ttf") format("truetype");
+}
+
+@font-face {
+	font-family: "Helvetica Bold";
+	src: url("../fonts/Helvetica/HelveticaLTStd-Comp.otf") format("opentype");
+}
+
+@font-face {
+	font-family: "Helvetica Lt";
+	src: url("../fonts/Helvetica/HelveticaLTStd-Light.otf") format("opentype");
+}
+
+@font-face {
+	font-family: "Open Sans";
+	src: url("../fonts/Open_Sans/OpenSans-Regular.ttf") format("truetype");
+}
+
+@font-face {
+	font-family: "Roboto";
+	src: url("../fonts/Roboto/Roboto-Regular.ttf") format("truetype");
+}
+
+@font-face {
+	font-family: "Titillium Web";
+	src: url("../fonts/Titillium_Web/TitilliumWeb-SemiBold.ttf") format("truetype");
+}
+
+/* Colors used for text and backgrounds */
+$brightblue-color: rgb(15, 87, 234);
+$lightblue-color: #65B6F0;
+$lightgray-border: 2px solid #E3E6E8;
+$lightgray-color: #E3E6E8;
+$lightred-color: rgb(237, 94, 104);
+$lightmidnight-color: rgb(54, 59, 89);
+$midnightblue-color: rgba(23, 28, 56, 1);
+$owgray-color: rgb(72, 72, 72);
+$nyred-color: rgba(252, 28, 38, 1);
+
+/* Colors that get pulled throughout the theme */
+$bg-color: #FFF;
+$bright-color: $nyred-color;
+$complementary-color: $brightblue-color;
+$solid-color: $midnightblue-color;
+
+/* Styles for different shadows */
+$box-shadow: 0 10px 6px -6px #777;
+$hard-shadow: 1px 2px 0 rgba(0, 0, 0, 1);
+$text-shadow: 1px 2px 1px rgba(0, 0, 0, 0.3);
+$transition: 0.2s ease-in-out;
+$soft-shadow: rgba(0, 0, 0, 0.06) 0 2px 4px 0;
+
+/* Font Variables */
+$body-font: "Titillium Web";
+$form-font: "Open Sans";
+$link-text: sans-serif;
+$nav-font: sans-serif;
+$title-font: "Circular";
+
+@mixin fullwidth() {
+	left: 50%;
+	margin-left: -50vw;
+	margin-right: -50vw;
+	max-width: none;
+	position: relative;
+	right: 50%;
+	width: 100vw;
+}
+
+@mixin btnBackground($color) {
+	background: $color;
+	border: 2px solid $color;
+	color: #FFF;
+}
+
+@mixin redBackground() {
```

Comment:
The variable should be more descriptive to define it's purpose rather than a color.  What if we decide we do not want red anymore, this SASS mixin would be invalid.